taxes_by_invoice/templates/detail_lines_macro.html

```diff
@@ -1,7 +1,7 @@
 {% set nc = namespace(before_invoice_id=None) %}
 {%for l in record %}
     {% if nc.before_invoice_id != l.invoice.raw.id %}
-        <tr>
+        <tr class="{% if l.invoice.raw.state == 'cancelled' %}grey{% endif %}">
```

Comment:
I think we should explain in the report header that cancelled invoices are shown in grey and that those with a cancel move are not added to the total.

Reply:
f2f4a6b40da0d3abeaeafeaccc89f056c1295abb

Add checklist the locales:
Las facturas canceladas se muestran en gris y aquellas con movimiento de cancelación no se suman al total.
Les factures cancel·lades es mostren en gris i les que tenen un moviment de cancel·lació no s'afegeixen al total.